yarn-project/kv-store/src/lmdb/store.ts

```diff
@@ -64,13 +65,12 @@ export class AztecLmdbStore implements AztecKVStore, AztecAsyncKVStore {
     ephemeral: boolean = false,
     log = createLogger('kv-store:lmdb'),
   ): AztecLmdbStore {
-    if (path) {
-      mkdirSync(path, { recursive: true });
-    }
+    const dbPath = path ?? join(tmpdir(), randomBytes(8).toString('hex'));
```

Comment:
I think we could make `path` required in order to catch any places that ask for a tmp location?

Reply:
There is a place in the node `server.ts` that requests a tmp store. I don't see that this needs to provide a location, it calls delete on the stores once it's finished. With this change, that call is now actually cleaning up the directory which is what would be expected.